http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSInterpreterSpec.scala

```diff
@@ -275,6 +304,62 @@ class AKSInterpreterSpec extends AnyFlatSpecLike with TestComponent with Leonard
     deletion.unsafeRunSync()(cats.effect.unsafe.IORuntime.global)
   }
 
+  it should "create and poll a wds app, then successfully delete it" in isolatedDbTest {
```

Comment:
Can you parameterize this test with the `it should "create and poll a coa app, then successfully delete it" ` test?

Or do the Scala spirits not allow that to be easily done here?